Makefile.armv7a

```diff
@@ -10,12 +10,16 @@ CROSS ?= arm-phoenix-
 
 CC := $(CROSS)gcc
 
+DEFAULT_OLVL := -O2
 ifeq ($(DEBUG), 1)
-  CFLAGS += -Og
+  OLVL := -Og
 else
-  CFLAGS += -O2 -DNDEBUG
+  OLVL := $(or $(OLVL),$(DEFAULT_OLVL))
```

Comment:
was wondering why not `?=` but I see it's always set because of filtering below. Not sure if there is a way to avoid replacing it with `?=` during some refactorization in the future so it's ok to leave it as-is

Reply:
?= does not work on defined but empty variable. 

So then after filtering I would have to undefining it if it;s empty. It also ok but i thought that this or will be more straightforward that way I do it.

I can change it if you think that undeifine in Makefile.common and ?= here was better